### PR TITLE
[Merged by Bors] - feat: lemmas about Finset.card

### PR DESCRIPTION
**Pull request by qawbecrdtey**

### CODE REVIEW

File: Mathlib/Data/Finset/Card.lean [hunk 1]
```diff
@@ -465,6 +465,10 @@ theorem card_sdiff_add_card : (s \ t).card + t.card = (s ∪ t).card := by
 lemma card_sdiff_comm (h : s.card = t.card) : (s \ t).card = (t \ s).card :=
   add_left_injective t.card $ by simp_rw [card_sdiff_add_card, ←h, card_sdiff_add_card, union_comm]
 
+theorem Finset.card_eq_card_sdiff_add_card_inter (s t : Finset α) :
+    s.card = (s \ t).card + (s ∩ t).card := by
+  rw [← card_disjoint_union (disjoint_sdiff_inter _ _), sdiff_union_inter]
```
Comment (YaelDillies) [inline Mathlib/Data/Finset/Card.lean:470]:
```suggestion
@[simp] lemma card_sdiff_add_card_inter (s t : Finset α) :
    (s \ t).card + (s ∩ t).card = s.card := by
  rw [← card_disjoint_union (disjoint_sdiff_inter _ _), sdiff_union_inter]
 
@[simp] lemma card_inter_add_card_sdiff (s t : Finset α) :
    (s ∩ t).card + (s \ t).card = s.card := by
  rw [add_comm, card_sdiff_add_card_inter]
```

File: Mathlib/Data/Finset/Card.lean [hunk 2]
```diff
@@ -465,6 +465,14 @@
 lemma card_sdiff_comm (h : s.card = t.card) : (s \ t).card = (t \ s).card :=
   add_left_injective t.card $ by simp_rw [card_sdiff_add_card, ←h, card_sdiff_add_card, union_comm]
 
+@[simp] lemma card_sdiff_add_card_inter (s t : Finset α) :
+    (s \ t).card + (s ∩ t).card = s.card := by
+  rw [← card_disjoint_union (disjoint_sdiff_inter _ _), sdiff_union_inter]
+
```
[the remark this answers is not in the document]
Comment (YaelDillies) [reply Mathlib/Data/Finset/Card.lean:471]: Sorry what?

[the remark this answers is not in the document]
Comment (qawbecrdtey) [reply Mathlib/Data/Finset/Card.lean]: It should work now

**Comment by ocfnash** [comment]
bors merge

**Comment by robertylewis** [comment]
bors merge

**Comment by bryangingechen** [comment]
bors merge

**Comment by robertylewis** [comment]
bors merge

**Pull request closed without merging**